### PR TITLE
Search dropdown bugfixes

### LINKED ISSUE #452: Landing page collection picker will not load correctly if starting the viewer outside of the landing page

### PR DESCRIPTION
**Pull request by jabrah**
Closes https://github.com/jhu-digital-manuscripts/rosa2/issues/452

Collection picker dropdown should initialize correctly when loading the viewer in a place not showing the collection picker.

### CODE REVIEW

File: css/aor/search-facets.css [hunk 1]
```diff
@@ -2,9 +2,9 @@
   margin-top: -1px;
 }
 .facet-container-scrollable {
-  position: absolute;
+  /* position: absolute;
```
Comment (jabrah) [inline css/aor/search-facets.css:5]: I think this change causes the facets to no longer be scrollable

Comment (jabrah) [reply css/aor/search-facets.css:5]: This change (especially the other CSS rules commented out, namely commenting out setting the bottom) unsets the static height of the element, moving the scroll responsibility to the parent container, which we don't want

**Comment by jabrah** [comment]
Changes deployed to https://rosetest.library.jhu.edu/dlmm/

**Comment by markpatton** [comment]
Only issue I notice is that while the manuscript list gets correctly initialized, the selection in the dropdown doesn't change. For example try https://rosetest.library.jhu.edu/dlmm/#pizan.

**Pull request merged**